pycsob/client.py

```diff
@@ -102,9 +102,6 @@ def payment_init(self, order_no, total_amount, return_url, description, merchant
         :return: response from gateway as OrderedDict
         """
 
-        if len(description) > 255:
```

Comment:
The parameter `description` is still used to create shopping cart contents, but is no longer sent directly, so this check is not needed anymore.